cli/Cargo.toml

```diff
@@ -126,3 +126,22 @@ pkg-url = "{ repo }/releases/download/v{ version }/jj-v{ version }-{ target }.{
 
 [lints]
 workspace = true
+
+[package.metadata.deb]
+name = "jj"
+maintainer = "Your Name <your@email.com>"
+copyright = "Copyright 2025 The Jujutsu Authors"
+license-file = "LICENSE"
+section = "vcs"
+priority = "optional"
+extended-description = "Distributed Version Control System inspired by Git, Mercurial and Darcs."
+assets = [
+    { source = "docs/*", dest = "usr/share/doc/jj/", mode = "644"},
+    { source = "docs/*/*", dest = "usr/share/doc/jj/", mode = "644"},
+    { source = "target/release/jj", dest = "usr/bin/", mode = "755"},
+    { source = "target/release/fake-diff-editor", dest = "usr/bin/", mode = "755"},
+    { source = "target/release/fake-editor", dest = "usr/bin/", mode = "755"},
+    { source = "target/release/fake-formatter", dest = "usr/bin/", mode = "755"},
```

Comment:
We don't need to add these to the package, do we? I think it would be strange to put these in the users `bin/` dir after installation. They're only needed for the test suite, anyway.

Reply:
All 3?

Reply:
Yes, we don't want any of these 3. Hopefully, this will mean that a plain `cargo deb` will work as well.

Reply:
Done

Reply:
Thanks!